### ISSUE TITLE
index.cjs.js requires non existent files

### ISSUE THREAD

**Issue by sccorby** [thanhtunguet/grapesjs-react#7]
Does not compile when used within a Create React App. 

Lines 5 and 6 of index.cjs.js require two files that do not exist. 
```
var helpers = require('helpers');
require('components/GComponent');
```
Are these modules located elsewhere? 

Thanks!

**Comment by msteinmn** [comment]
Same issue:
./node_modules/grapesjs-react/dist/index.cjs.js
Module not found: Can't resolve 'components/GComponent'

**Comment by thanhtunguet** [comment]
> Does not compile when used within a Create React App.
> 
> Lines 5 and 6 of index.cjs.js require two files that do not exist.
> 
> ```
> var helpers = require('helpers');
> require('components/GComponent');
> ```
> Are these modules located elsewhere?
> 
> Thanks!

Thank you for reporting this issue.
This is caused because I used absolute import without absolute import plugin.

I have created a fix for it and published in version 1.0.3.
Please update your dependencies.
